pytorch_lightning/callbacks/progress.py

```diff
@@ -387,8 +388,14 @@ def _update_bar(self, bar):
             bar.update(delta)
 
 
-def convert_inf(x):
+def convert_inf(x: Union[int, float]) -> Optional[int]:
```

Comment:
```suggestion
def convert_inf(x: Optional[Any]) -> Optional[Any]:
```

Reply:
You could make an argument that you want to tighten the definition to the expected use (not the allowed). But in that case, it'd be

```suggestion
def convert_inf(x: Optional[Union[int, float]]) -> Optional[Union[int, float]]:
```

Reply:
ok, I think I need to add imports